random-menu.rb

```diff
@@ -0,0 +1,36 @@
+adjectives = ["hot", "soft", "cold", "creamy", "green", "sour",
+"salty", "peppered", "sweet", "delicious"]
+cooking_styles = ["pan-fried", "steamed","boiled", "fried","baked", "stewed",
+  "sauted", "roasted","smoked", "deep-fried"]
+food = ["pizza", "cake","dumplings", "oatmeal","egg", "turkey","pasta",
+  "burger", "sandwich", "salad"]
+
+input = 0
+stop = false
+until stop # Ask user for input until he/she enters number less than 10
+  puts "How many items would you like to see?"
+  input = gets.chomp.to_i
+  if input <= 10
+    stop = true
+  end
+end
+
+new_array = [] # To store new randomly-generated food items
```

Comment:
`new_array` is not a particularly descriptive variable name. Can you think of something clearer to call it?